.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
do we need this file?

Reply:
> do we need this file?

i guess so, since we have tsconfig.json renamed to tsconfig.base.json, this will tell eslint to read that config file for linting ts files. or do you have any advice?